src/Parallel/PhaseControl/CheckpointAndExitAfterWallclock.hpp

```diff
@@ -235,6 +235,10 @@ CheckpointAndExitAfterWallclock::arbitrate_phase_change_impl(
     } else {
       // if current_phase is WriteCheckpoint, we follow with updating options
       if (current_phase == Parallel::Phase::WriteCheckpoint) {
+        const auto now = std::chrono::system_clock::now();
+        const std::time_t now_c = std::chrono::system_clock::to_time_t(now);
+        Parallel::printf("Restarting from checkpoint. Date and time: %s\n",
+                         std::ctime(&now_c));
```

Comment:
Please print time in UTC. Dealing with time zones has been very challenging with SpEC output.

Reply:
ok. even better would be (also) storing the timing in some data file, rather than the log file. not sure where yet though (I was thinking reductions file attributes).

Reply:
Yea, that's a good idea, but I'm also not sure where. We'll need to update the startup info we print (usually into spectre.out) anyway to include things like Slurm info. I  did this for spec and will port it to spectre

Reply:
Maybe we just parse timing etc out of the log file in a postprocessing step when a simulation is complete, and store in a yaml or json metadata file that will then interface with the SXS catalog.

Reply:
Btw, Charm currently outputs date and time at startup, but probably not in UTC. So when you update the startup info, also add the UTC date and time please.